libraries/standard/json/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Include filepaths for source and include.
+include(jsonFilePaths.cmake)
+
+# JSON library target.
+add_library( json
+             ${JSON_SOURCES} )
+
+# JSON public include path.
+target_include_directories( json PUBLIC ${JSON_INCLUDE_PUBLIC_DIRS}
+                                        ${LOGGING_INCLUDE_DIRS} )
```

Comment:
The JSON library doesn't have any logging calls in it, so there shouldn't be this dependency

Reply:
Thanks